taxcalc/reforms/TCJA.json

```diff
@@ -153,5 +153,40 @@
                             "2026": 0},
     "ID_Medical_frt": {"2017": 0.075,
                        "2019": 0.075},
-    "parameter_indexing_CPI_offset": {"2017": -0.0025}  
+    "parameter_indexing_CPI_offset": {"2017": -0.0025},
+
+    // extra parameters:
+    "II_em_ps": {"2018": [9e99, 9e99, 9e99, 9e99, 9e99]},
+    "STD_Dep": {"2018": 1050, "2019": 1100},
+    "STD_Aged": {"2018": [1600, 1300, 1300, 1600, 1300],
+                 "2019": [1650, 1300, 1300, 1650, 1300]},
+    "CG_brk1": {"2018": [38600, 77200, 38600, 51700, 77200],
+                "2019": [39375, 78750, 39375, 52750, 78750]},
+    "CG_brk2": {"2018": [425800, 479000, 239500, 452400, 479000],
+                "2019": [434550, 488850, 244425, 461700, 488850]},
+    "AMT_CG_brk1": {"2018": [38600, 77200, 38600, 51700, 77200],
+                    "2019": [39375, 78750, 39375, 52750, 78750]},
+    "AMT_CG_brk2": {"2018": [425800, 479000, 239500, 452400, 479000],
+                    "2019": [434550, 488850, 244425, 461700, 488850]},
+    "AMT_child_em": {"2018": 7600,
+                     "2019": 0},
+    "AMT_brk1": {"2018": 191100,
+                 "2019": 194800},
+    // Only adding 2018, 2019 values for now. Could add 2020, 2021
+    "EITC_c": {"2018": [519, 3461, 5716, 6431],
+               "2019": [529, 3526, 5828, 6557]},
+    "EITC_ps": {"2018": [8490, 18660, 18660, 18660],
+                "2019": [8650, 19030, 19030, 19030]},
+    "EITC_ps_MarriedJ": {"2018": [5680, 5690, 5690, 5690],
+                         "2019": [5800, 5790, 5790, 5790]},
+    "EITC_InvestIncome_c": {"2018": 3500,
+                            "2019": 3600},
+    "ETC_pe_Single": {"2018": 67,
+                      "2019": 68},
+    "ETC_pe_Married": {"2018": 134,
+                       "2019": 136},
+    "FST_AGI_thd_lo": {"2018": [1000000, 1000000,  500000, 1000000, 1000000],
+                       "2019": [1000000, 1000000,  500000, 1000000, 1000000]},
+    "FST_AGI_thd_hi": {"2018": [2000000, 2000000, 1000000, 2000000, 2000000],
+                       "2019": [2000000, 2000000, 1000000, 2000000, 2000000]}
```

Comment:
@jdebacker in light of your [comment](https://github.com/PSLmodels/Tax-Calculator/issues/2542#issuecomment-888557463), where should I put these values since they are not really related to the TCJA reform?

Reply:
@hdoupe I think that:
1. `"EITC_ps_MarriedJ": {"2017": [5600, 5600, 5600, 5600]}` should be removed from `2017_law.json`
2.  You should not have to add anything under `// extra parameters:` -- all these values should be the same in current law and should not be changed when reverting law using `2017_law.json`.

Can you tell me if things work with the above changes?

Reply:
> You should not have to add anything under `// extra parameters`: -- all these values should be the same in current law and should not be changed when reverting law using `2017_law.json`.

I think I'm missing something because these values are all modified in `2017_law.json`. For example, `II_em_ps` is updated in `2017_law.json`: https://github.com/PSLmodels/Tax-Calculator/blob/2ccfcb446d281aa8f733be59f591de931a41ad24/taxcalc/reforms/2017_law.json#L86

But it is not updated in `TCJA.json`. So there is no way for it to go back to current law after being changed in `2017_law.json` unless it is added to `TCJA.json` or updated some other way.

Reply:
So, if I remove these values from `TCJA.json`, the tests fail. This is because the values are changed at the beginning of the test when the `2017_law.json` file is applied, but they are never changed back for 2018 and 2019. Some of them like `EITC_c` are changed but only later in 2021 and 2022 with the ARPA updates.

Reply:
> So, if I remove these values from TCJA.json, the tests fail. This is because the values are changed at the beginning of the test when the 2017_law.json file is applied, but they are never changed back for 2018 and 2019. 

Yes, but as noted in Issue #2542, most of the values that are changes in 2017_law.json are set to the same values as in `policy_current_law.json`.